### DISCUSSION TITLE
[STUDY GROUP]: GitHub Foundations Exam

### DISCUSSION THREAD

**Discussion by v-fidelusaleksander** [v-fidelusaleksander/ghcertified#166] [Study Together!]
# GitHub Foundations :book: STUDY GROUP

Hello everyone,

Many of us are preparing for the GitHub Certification Exams and I thought it would be beneficial to start a discussion where we can share:

-  experiances
-  study tips
-  ask questions 
- anything else regarding any of the topics covered in the exam 

Looking forward to hearing from you all! :rocket:

**Comment by raptorrico** [comment]
Trying to judge my level of readiness for the real exam. Are these questions of a similar difficulty to the exam? I scored 80/101 on my first attempt.

**Reply by v-fidelusaleksander** [reply]
I think the questions are of similar difficulty and maybe even the ones on ghcertified a bit harder.

For all my exams I felt ready when I knew I could answer/talk about every or almost every topic in the study guides.

Let us know if you've passed and with what score!

**Reply by raptorrico** [reply]
I did pass, just! 63%

I have no idea how that's a pass mark though, I thought the minimum would be around 70% But a pass is a pass.

**Reply by v-fidelusaleksander** [reply]
Yeah 63 seems low for a pass

But a pass is a pass :) Congrats!

**Comment by eduardconstantin** [comment]
Great set of questions. I scored over 90 in my 5-6 attempts.
I will try to pass the exam this Friday.

**Reply by v-fidelusaleksander** [reply]
Good luck on your exam! 🚀📚

Let us know if was easier or harder than the practice exams!

**Reply by eduardconstantin** [reply]
I passed it! 
<img width="391" alt="image" src="https://github.com/FidelusAleksander/ghcertified/assets/9199763/5adc78bd-ed2b-44d9-b09c-c04c1b923612">
I think I found 2-3 questions that were pretty similar to what you have. Some were trickier and some easier.

**Reply by v-fidelusaleksander** [reply]
Congratulations! Now go for the other ones :)

**Reply by eduardconstantin** [reply]
Thank you! I am thinking to get Actions too, but later this year

**Comment by mattoliveira** [comment]
Tomorrow is my exam and during my study journey I took 6 practice exams, I passed 5 of them, I only failed in my first practice exam...the last one I took were 109 questions from this repository and I scored 95/109

I'll be back tomorrow to let you know if I passed or not.

**Reply by mattoliveira** [reply]
I passed! I got 78% 🙏🏻

**Reply by rilindo** [reply]
Congrats!

**Reply by v-fidelusaleksander** [reply]
Congrats! What's your opinion on the exam? @mattoliveira

**Reply by mattoliveira** [reply]
@FidelusAleksander The questions were well aligned with those in this repository, for sure i got some harder and others easier, but overall very similar

**Comment by Salgado2004** [comment]
Hello guys, my GHF exam is in one week and I'm studying with the MS learning path and doing practice exams. Do you have any advice to improve my knowledge for the exam?

**Reply by mattoliveira** [reply]
I recommend to watch videos about topics you have difficulty

I used GHF course on ExamPro (there is free option) https://www.exampro.co/github-foundations

**Reply by Salgado2004** [reply]
To update you guys... I passed my exam with 88%! 🎉

Everything worked fine... the questions of the practice exam were indeed pretty similar to the actual exam. The GHF course on ExamPro also helped me to understand some concepts. Thanks for the advice!

**Reply by v-fidelusaleksander** [reply]
Congratulations @Salgado2004 🎉🚀

**Comment by CarlosJoseChaconChavarria** [comment]
> I recommend to watch videos about topics you have difficulty
> 
> I used GHF course on ExamPro (there is free option) https://www.exampro.co/github-foundations

did you paid for all other exams? I know exampro has 1 free but I am not sure if paying those extra $$$ will help me pass

thanks

**Reply by mattoliveira** [reply]
> > I recommend to watch videos about topics you have difficulty
> 
> > 
> 
> > I used GHF course on ExamPro (there is free option) https://www.exampro.co/github-foundations
> 
> 
> 
> did you paid for all other exams? I know exampro has 1 free but I am not sure if paying those extra $$$ will help me pass
> 
> 
> 
> thanks

I only did the free option, it was enough for me

**Reply by CarlosJoseChaconChavarria** [reply]
glad to hear that I am going to take this too

**Comment by trentonyo** [comment]
Hello fellow learners! 

I was awarded my Foundations cert yesterday, and I can definitely thank ghcertified for getting me there! 

The questions were comparable in difficulty, and there were only a few questions that stumped me and I don't recall seeing on the ghcertified practice exam. 

I had 75 questions, and passed with 49/60 points 🥦

**Comment by codess-aus** [comment]
Hey everyone, I haven't sat any yet but am wondering about the exam environment. Is it strictly proctored like the Microsoft exams or can I just sit it wherever I am. I'm worried that I don't have a clean room, without papers in it.

**Reply by v-fidelusaleksander** [reply]
Ive personally taken all my GitHub certs in an exam center for this reason. I think they use the same company for the certificates (PSI) so I would imagine it's the same issues and concerns

**Reply by Salgado2004** [reply]
I've taken online exams for Microsoft and GitHub and I (personally) felt that Microsoft was a little bit more restrict. For the same environment that I had, the proctors asked to remove more itens from my desk (even my pen). But GitHub also asked to clean the environment and show pictures of all four corners of the room.

**Reply by kamakiJr** [reply]
I have taken the examination at home; it was a hassle to unplug my monitors and clean up the work area. But I was comfortable testing from my office.

**Comment by rzavalik** [comment]
Q23: What is the minimum permission a user must have to directly push commits to a repository owned by an organization?

- [ ] Delete the repository
- [ ] Change repository settings
- [ ] Manage issues and pull requests
- [ ] Push commits
- [ ] Create new repositories

The quiz is saying the correct one is `Manage issues and pull requests` and not `Push commits`... do you agree with it? Is that really correct?

**Reply by pratikshac1** [reply]
I think it should be push commits.

**Comment by JellyfishManiac** [comment]
I just wanted to write saying that I am going to be taking the exam shorty here. I finished up with the Microsoft Learn pathways for the Foundations Exam (both parts) and am going to do these practice questions next. Any advice or motivation anyone can give me? I am super excited here!!!

**Reply by v-fidelusaleksander** [reply]
Good luck on your exam!

**Comment by JellyfishManiac** [comment]
Update everyone: I just passed!!! Got 700 which is passing. So hyped. Thank you so much to everyone for the awesome resources. As an aspiring Computer Engineer I am forever grateful for this experience.

Of note I was not able to get a voucher for a student discount due to the switch to Microsoft, but it was still worth it to me.

**Comment by mevrouwdewi** [comment]
Just wanted to post my appreciation for this repo and the question bank. I'm usually wary of self-made practice exams, but the quality of these questions is top notch and I'm feeling more confident for my exam tomorrow. :)

**Reply by JellyfishManiac** [reply]
Wish you luck! 👍

**Comment by kamakiJr** [comment]
I just passed the Foundations certification. I started my studies mid-September. 
I setup a repo and anytime that I ran into a question or example, I would apply the experience in my repo.
I took the **ghcertified** practice exam two times. 

While the actual exam appeared more difficult than the practice exam, I was able to score over 860.
